playwright-e2e/fixtures/singular-fixture.ts

```diff
@@ -28,11 +28,14 @@ const fixture = base.extend<Fixtures>({
     const homePage = await launchHomePage(page);
     await use(homePage);
   },
-  dashboardPage: async ({ page }, use) => {
-    const homePage = await launchHomePage(page);
+
+  dashboardPage: async ({ page, homePage }, use) => {
     await homePage.signUp();
     await new PreScreeningPage(page).enterInformationAboutYourself();
-    await auth.createAccountWithEmailAlias(page, { email: SINGULAR_USER_EMAIL, password: SINGULAR_USER_PASSWORD });
+    process.env.SINGULAR_USER_EMAIL_ALIAS = await auth.createAccountWithEmailAlias(page, {
+      email: SINGULAR_USER_EMAIL,
+      password: SINGULAR_USER_PASSWORD
+    });
```

Comment:
Store generated email alias in `process.env.SINGULAR_USER_EMAIL_ALIAS` to be used in `account-login.spec.ts`.